### PR TITLE
fix: [SRM-14595]: No matching data for the applied search page shows up if the there are no results for search option on SLO listing page

### PR DESCRIPTION
**Pull request by karansaraswat19**
#### Screenshots

NA

#### PR Checklist

<details>
<summary>Please check if your PR fulfils the following requirements</summary>

- [ ] Tests for the changes have been added. Ideally, include a test that fails without this PR but passes with it.
- [ ] Docs have been [added/updated](https://harness.atlassian.net/jira/software/c/projects/DOC/boards/40).
</details>

<details>
<summary>Use the following comments to re-trigger PR Checks</summary>

- Jest: `retrigger jest`
- Prettier: `retrigger prettier`
- Type Check: `retrigger typecheck`
- ESLint: `retrigger eslint`
- Sonar: `retrigger sonar`
- Standards: `retrigger standards`
- Build: `retrigger build`
- Title Check: `retrigger titlecheck`
- Feature Name Check: `trigger featurenamecheck`
- Coverage: `retrigger coverage`
- Rebase: `trigger rebase`
- Cypress Rest: `retrigger cypress-rest`
- Cypress CD: `retrigger cypress-cd`
- Cypress Pipeline: `retrigger cypress-pipeline`
- Cypress CV: `retrigger cypress-cv`
- Fix Prettier: `fix prettier`
</details>

### CODE REVIEW

**Comment by karansaraswat19** [comment]
retrigger titlecheck

**Review by deepesh-ui — COMMENTED** [review]
Please add test case regarding this

**Review by praneshg239 — APPROVED** [review]

File: src/modules/85-cv/pages/slos/__tests__/CVSLOListingPage.utils.test.ts [hunk 1]
```diff
@@ -46,4 +65,18 @@ describe('CVSLOListingPage utils', () => {
...
     expect(result).toEqual(riskCountQueryParamsWithAllMonitoredServiceResult)
   })
+
+  test('isSLOFilterApplied should return correct value when no filter or at least one filter is applied', () => {
+    // when no filter applied
+    let result = isSLOFilterApplied(getString, filterState)
+    expect(result).toBe(false)
+
+    // when MS selected
+    result = isSLOFilterApplied(getString, { ...filterState, monitoredService: { label: 'msdemo', value: 'msdemo' } })
```
Comment (deepesh-ui) [inline src/modules/85-cv/pages/slos/__tests__/CVSLOListingPage.utils.test.ts:75]: return type should consistent for the function, why we sending  string here ?

Comment (karansaraswat19) [reply src/modules/85-cv/pages/slos/__tests__/CVSLOListingPage.utils.test.ts:75]: fixed, now the function will return boolean values

**Review by deepesh-ui — APPROVED** [review]

**Comment by karansaraswat19** [comment]
retrigger cypress-cd

**Comment by karansaraswat19** [comment]
retrigger cypress-cd

**Comment by karansaraswat19** [comment]
retrigger cypress-pipeline

**Comment by karansaraswat19** [comment]
retrigger cypress-cv

**Comment by karansaraswat19** [comment]
retrigger cypress-pipeline

**Review by praneshg239 — APPROVED** [review]

**Pull request merged**